src/App.tsx

```diff
@@ -1,14 +1,65 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, {
+  useCallback, useEffect, useMemo, useState,
+} from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos, getUser } from './api';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [currentTodo, setCurrentTodo] = useState<Todo | undefined>();
+  const [currentUser, setCurrentUser] = useState<User | undefined>();
```

Comment:
You should fetch this user in `TodoModal`

Reply:
I really want to do this. However, with this approach, I get nothing but errors.
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/55781950/10571d78-53d8-48ef-9527-5bac45a2e910)

Not to mention the tests that did pass saw a hefty increase in execution time, I am not entirely sure why.